PaArray[] pa2dArray
IntPoint  min
IntPoint  max
